java/spacewalk-java.changes

```diff
@@ -1,3 +1,5 @@
+- Hide useless fields for containerized proxies in UI
```

Comment:
If you could add a simple screenshot in the PR of what is left in the UI it would be nice, just for completeness. Thanks :wink: